packages/slider/README.md

```diff
@@ -115,13 +115,30 @@ for a multi-handle slider, you can format the combined value label for all
 handles by passing a formatting function to the `getAriaValueText` property
 on the parent `sp-slider`.
 
-You can suppress the value label altogether using the `hide-value-label`
-attribute.
+### Label Visibility
+
+Be default an `<sp-slider>` element has both a "text" label and a "value" label. Either or both of these can be surpressed visually as needed by your application UI. This delivery is controlled by the `label-visibility` attribute (or `labelVisibility` property) which accepts `text`, `value`, or `none` as values.
+
+Use `label-visibility="text"` to supress the "value" label:
+
+```html
+<sp-slider label="No visible value label" label-visibility="text"></sp-slider>
+```
+
+Use `label-visibility="value"` to supress the "text" label:
 
 ```html
-<sp-slider label="No value label" hide-value-label></sp-slider>
+<sp-slider label="No visible text label" label-visibility="text"></sp-slider>
```

Comment:
Shouldn't this be 
```suggestion
<sp-slider label="No value label" label-visibility="text"></sp-slider>
```

By explicitly saying you want the `text` label, you are implicitly saying you don't want the value label...right?

Reply:
Yes, this _IS_ incorrect. As per https://github.com/adobe/spectrum-web-components/pull/1684/files#diff-1343cd8d1ec716e47c2b3da1c60bdfd6c44f4935d88ec402d346b762e852ffa1R128 in the other direction, however.